inc/extras.php

```diff
@@ -166,6 +166,22 @@ function responsive_widget_counts( $params ) {
 		return $params;
 	}
 
+	/**
+	 * Filters the $is_widget_empty variable.
+	 *
+	 * @since 2.1.9
+	 *
+	 * @params bool $is_widget_empty The empty/full status of the widget content.
+	 *
+	 * @params array $params An array of widget options.
+	 */
+	$is_widget_empty = apply_filters( 'responsive_is_widget_empty', $is_widget_empty = false, $params );
```

Comment:
Do we want to specify `$is_bu_widget_empty`, or do you think a generic name would be fine here?

Reply:
I chose to be more generic there. The filter can be used for any type of widget, not just BU widgets.